2021/day10/day10.java

```diff
@@ -0,0 +1,108 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedList;
+
+public class day10 {
+    public static class Scores {
+        int corruptedScore;
+        Long incompleteScore;
+
+        public Scores(int x, Long y) {
+            corruptedScore = x;
+            incompleteScore = y;
+        }
+    }
```

Comment:
Now that I think about it creating class Scores doesn't make that much sense.
I think that what makes sense to me is creating Line class that has 

>String chunks 
>int/long score
>boolean isCorrupted

and building the rest from there.
The only issue I have with that approach is that the code used to calculate the score and to decide if line is corrupted or not is "the same code" and it would end up being doubled.

Reply:
doubled = I would have the same six/seven lines in two (or even three) places